lib/core/MongoDbOperationStore.ts

```diff
@@ -101,24 +99,41 @@ export default class MongoDbOperationStore implements IOperationStore {
    */
   public async delete (transactionNumber?: number): Promise<void> {
     if (transactionNumber) {
-      await this.collection!.deleteMany({ transactionNumber: { $gt: Long.fromNumber(transactionNumber) } });
+      await this.collection!.deleteMany({ txnNumber: { $gt: Long.fromNumber(transactionNumber) } });
     } else {
       await this.collection!.deleteMany({});
     }
   }
 
+  public async deleteUpdatesEarlierThan (didUniqueSuffix: string, transactionNumber: number, operationIndex: number): Promise<void> {
```

Comment:
I don't think so because:

1. The caller (`Resolver`) would have to construct an instance of `NamedAnchoredOperationModel` which it doesn't already have.
1. The caller would be passing unnecessary info like `OperationType` and `operationBuffer` for the sake for conforming to the method signature.